libraries/serverHelpers/toolboxEdgeProxyRequestHandler.js

```diff
@@ -0,0 +1,15 @@
+const https = require('https');
+
+const toolboxEdgeProxyRequestHandler = (req, res) => {
```

Comment:
It would be good to have documentation on what req and res are expected to be and what format is expected from req's params

Reply:
These are the same params that Express passes on to its handlers, since it's being used as a handler on some Express routes